### PR TITLE
MAR-4612: Upgrade to Python 3

### PR DESCRIPTION
**Pull request by Reshma-M-A**

### CODE REVIEW

**Review by Revit — APPROVED** [review]

File: app/__init__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import os
-import urlparse
+import urllib
```
Comment (Revit) [inline app/__init__.py:2]: Related to line 77 , should import as 'import urllib.parse'

File: app/main/helpers/services.py [hunk 2]
```diff
@@ -5,10 +5,7 @@
 
 from dmapiclient import APIError
 
-try:
-    import urlparse
-except ImportError:
-    import urllib.parse as urlparse
+import urllib
```
Comment (Revit) [inline app/main/helpers/services.py:8]: related to line 54 and 55 , should import as 'import 'urllib.parse'

**Comment by Reshma-M-A** [comment]
So far looks good!

File: app/main/views/briefs.py [hunk 3]
```diff
@@ -116,8 +115,8 @@ def send_thank_you_email_to_responders(brief, brief_response, brief_response_url
     ess = brief.get('essentialRequirements', [])
     nth = brief.get('niceToHaveRequirements', [])
 
-    ess = zip(ess, brief_response.get('essentialRequirements', []))
-    nth = zip(nth, brief_response.get('niceToHaveRequirements', []))
+    ess = list(zip(ess, brief_response.get('essentialRequirements', [])))
+    nth = list(zip(nth, brief_response.get('niceToHaveRequirements', [])))
```
Comment (Revit) [inline app/main/views/briefs.py:120]: Dont understand the current codes with this:
zip(ess, brief_response.get('essentialRequirements', []))

Same thing is inserted into zip twice.

File: tests/app/main/test_login.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # coding: utf-8
-from __future__ import unicode_literals
 
-import urllib2
+import urllib
```
Comment (Revit) [inline tests/app/main/test_login.py:3]: should be import urllib.parse related to line 267

File: app/main/views/briefs.py (hunk 3 above)
Comment (tobiasbrown) [reply app/main/views/briefs.py:120]: This is getting the essential requirements and the nice to have requirements from a seller's response.  We need two statements here.

**Review by tobiasbrown — APPROVED** [review]

**Review by Reshma-M-A — COMMENTED** [review]
Changes looks good!

**Pull request merged**